python/dgl/nn/pytorch/conv/ginconv.py

```diff
@@ -98,6 +107,10 @@ def forward(self, graph, feat):
             :math:`(N_{in}, D_{in})` and :math:`(N_{out}, D_{in})`.
             If ``apply_func`` is not None, :math:`D_{in}` should
             fit the input dimensionality requirement of ``apply_func``.
+        edge_weight : torch.Tensor, optional
+            Optional tensor on the edge. If given, the convolution will weight
+            with regard to the edge feature. The shape is expected to be
+            :math:`(E_{edge}, 1)`.
```

Comment:
Not necessary to be `(E_{edge}, 1)`, it can be of any shape that broadcastable with `feat`.

Reply:
OK we can also support that, just then need to doc it more carefully.